docs/developer/code_standard_and_dev_guide.rst

```diff
@@ -61,3 +61,9 @@ The `[dev]` option will help you to install some related packages when developin
 .. code-block:: bash
 
     pip install -e .[dev]
+
+If you use zsh instead of bash, remember to use the following command.
+
+.. code-block:: bash
+
+    pip install -e ".[dev]"
```

Comment:
You can simply keep this case  which can handle all the cases in both zsh & bash .